index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="Top">
+            <a href=""><img src="images/MyLogo.png" alt="My Logo" ></a>
```

Comment:
- [ ] Please give some padding to the logo and the hamburger icon. They are so close to the page left and right margins.

Your Design |
:---------------:|
![a1](https://user-images.githubusercontent.com/39523312/171442018-338bb147-317b-48ef-895f-8d9a370b3c96.png)